src/main/resources/kobiton/ExecuteScript.py

```diff
@@ -0,0 +1,80 @@
+import urllib2
+import json
+import copy
+
+username = kobitonServer['username']
+apiKey = kobitonServer['apiKey']
+
+remoteServer = kobitonServer['remoteServer']
+
+defaultDeviceOrientation = kobitonServer['deviceOrientation']
+defaultCaptureScreenshots = kobitonServer['captureScreenshots']
+
+EXECUTE_TEST_URL = remoteServer + '/submit'
+
+def merge_devices():
+    result = []
+    for udid in privateDevices:
+        result.append({
+            'udid': udid
+        })
+
+    for id in kobiDevices:
+        params = kobiDevices[id].split(' | ')
+        if params[3] == 'cloudDevices':
```

Comment:
Don't need this, I think we just need to split then merge the devices to array. 
Just have one concerns what happen with duplicate udid of private devices of input field and get from task above?